cw/tvla.py

```diff
@@ -371,17 +371,17 @@ def run_tvla(ctx: typer.Context):
                     force=True,)
 
     if (cfg["mode"] != "kmac" and cfg["mode"] != "aes" and cfg["mode"] != "sha3" and
-            cfg["mode"] != "otbn"):
+            cfg["mode"] != "otbn" and cfg["mode"] != "otbn_modinv"):
```

Comment:
Hmm I also had this in mind, but then I thought, depending on how many future OTBN apps will be added, it could be a good idea to only add individual modes if necessary and keep a general `otbn` mode for everything else. Otherwise we might get too many modes, which would make the code even more complex.